app/src/main/java/com/h/pixeldroid/objects/Status.kt

```diff
@@ -99,9 +99,9 @@ data class Status(
     }
 
     fun getUsername() : CharSequence {
-        var name = account.display_name
-        if (name.isNullOrEmpty()) {
-            name = account.username
+        var name = account.username
+        if (name.isEmpty()) {
```

Comment:
Shouldn't this be kept as `isNullOrEmpty()`?

Reply:
Yep